app/views/collections/_show_document_list_menu.html.erb

```diff
@@ -2,18 +2,12 @@
   <button class="btn btn-sm dropdown-toggle" data-toggle="dropdown">Select an action <span class="caret"></span>
   </button>
   <ul class="dropdown-menu dropdown-menu-right">
+    
     <li>
-      <%= link_to raw('<i class="glyphicon glyphicon-link over"></i> Single-Use Link to File'), '#',
-                  class: "copypaste itemicon itemcode", title: "Single-Use Link to File", id: "copy_link_#{id}" %>
-    </li>
-    <li>
-      <%= link_to raw('<i class="glyphicon glyphicon-pencil"></i> Edit File'), sufia.edit_generic_file_path(id),
-                  class: 'itemicon itemedit', title: 'Edit File' %>
-    </li>
-    <li>
-      <%= link_to raw('<i class="glyphicon glyphicon-download-alt"></i> Download File'), sufia.download_path(id),
-                  class: 'itemicon itemdownload', title: 'Download File', target: '_new' %>
+      <%= link_to raw('<i class="glyphicon glyphicon-pencil"></i> Edit Work'), sufia.edit_generic_work_path(id),
```

Comment:
@tingtingjh It would be really nice if the label was I18N instead of hard coded.  The wireframe show the label as just Edit (remove Work)